core/unix/signal.c

```diff
@@ -3644,7 +3644,6 @@ convert_frame_to_nonrt(dcontext_t *dcontext, int sig, sigframe_rt_t *f_old,
     f_new->sig_noclobber = f_new->sig;
 #    elif defined(ARM)
     memcpy(&f_new->uc, &f_old->uc, sizeof(f_new->uc));
-    memcpy(f_new->retcode, f_old->retcode, sizeof(f_new->retcode));
```

Comment:
I think AArch32 does have retcode: I see it in https://elixir.bootlin.com/linux/latest/source/arch/arm/kernel/signal.c#L402

Reply:
I'm very sorry for misreading this, yes, AArch32 has a retcode.